config/permissions/api/custom.php

```diff
@@ -0,0 +1,65 @@
+
+<?php
+
+// TODO There are some permissions that seem like they should be tied
+// to models, but there is not an existing model for these permissions.
+// Figure out if we'll need to go ahead and create these models
+return [
+    'auth_profile.edit',
+    'user_management.access',
+    'permission.create',
+    'permission.edit',
+    'permission.show',
+    'permission.delete',
+    'permission.access',
+    'role.create',
+    'role.edit',
+    'role.show',
+    'role.delete',
+    'role.access',
```

Comment:
Curious if you think these could be deleted now, as they should be covered by the Model trait?